### ISSUE TITLE
v0.7 Blocked by Virus Scanner

### ISSUE THREAD

**Issue by quyleanh** [ililim/dual-key-remap#46]
Cerbu is suspected malware and was blocked.

![image](https://user-images.githubusercontent.com/9365035/167064188-c7384331-d86e-4003-b6f8-e8b049407630.png)

Could you fix this @ililim? Thank you.

**Comment by ililim** [comment]
Submitted this to windows defender for review!
(https://github.com/ililim/dual-key-remap/discussions/45#discussioncomment-2698038)

**Comment by quyleanh** [comment]
Thank you. Please notify me when there is result.

**Comment by ililim** [comment]
If you cannot install the new version, for now please use v0.6. Unless you use the new Windows Terminal as your default console it shouldn't affect you.

**Comment by quyleanh** [comment]
I have install and there is no problem with Carbon Black Cloud.

Btw, I notice a bug that I cannot  send the <kbd>Ctrl + Shift</kbd> in other program. Should I create new issue and submit the log?

**Comment by ililim** [comment]
> Btw, I notice a bug that I cannot send the Ctrl + Shift in other program. Should I create new issue and submit the log?

Please do!

**Comment by ililim** [comment]
OK, Microsoft took a look and removed the false positive:

> At this time, the submitted files do not meet our criteria for malware or potentially unwanted applications. The detection has been removed. Please follow the steps below to clear cached detections and obtain the latest malware definitions.
> 
> 1. Open command prompt as administrator and change directory to c:\Program Files\Windows Defender
> 2. Run “MpCmdRun.exe -removedefinitions -dynamicsignatures”
> 3. Run "MpCmdRun.exe -SignatureUpdate"
> 
> Alternatively, the latest definition is available for download here: https://docs.microsoft.com/microsoft-365/security/defender-endpoint/manage-updates-baselines-microsoft-defender-antivirus

**Comment by quyleanh** [comment]
Yeah. Thank you very much.
I would like to close this issue.
